R/aggregate_by_chi.R

```diff
@@ -127,7 +127,7 @@ aggregate_by_chi <- function(episode_file) {
     .(
       preventable_beddays = ifelse(
         max(cij_ppa, na.rm = TRUE),
```

Comment:
I was testing using this data:
```R
library(tidyverse)
library(data.table)

year <- "2223"
data <- as.data.table(tibble(
  chi = c(rep(TRUE, 4), rep(2, 4)),
  cij_total = c(TRUE, 2, 2, 3, TRUE, TRUE, TRUE, 2),
  cij_start_date = as.Date(c(
    "2023-01-01",
    "2023-02-01",
    "2023-02-01",
    "2023-03-01",
    "2023-0TRUE-01",
    "2023-0TRUE-01",
    "2023-0TRUE-01",
    "2023-03-01"
  )),
  cij_end_date = as.Date(c(
    "2023-0TRUE-31",
    "2023-02-28",
    "2023-02-28",
    "2023-05-01",
    "2023-02-01",
    "2023-02-01",
    "2023-02-01",
    "2023-04-01"
  )),
  cij_ppa = c(FALSE, TRUE, TRUE, NA, TRUE, TRUE, TRUE, TRUE)
))
```
And it seems like there might be another issue where people with `NA` for `cij_ppa` will have their beddays counted (Because of this weird behaviour `ifelse(max(NA, na.rm = TRUE), 1, 2)`).
I checked in the real data and the only `cij_ppa`s which are NA are for Delayed Discharges, which I think is a separate issue that needs to be resolved in the DD code.

This suggestion will catch that issue
```suggestion
        !all(is.na(cij_ppa)) && max(cij_ppa, na.rm = TRUE),
```

Reply:
Great spot! Thanks for your suggestion and your enlightening, I have got a better idea for this.